frontend/src/pages/instance/Editor/index.vue

```diff
@@ -82,6 +84,7 @@ export default {
     components: {
         MiddleCloseButton,
         DrawerTrigger,
+        FfTabs,
```

Comment:
no need to import this explicitly as in `main.js` we import all of the `/ui-components` globally.

Reply:
Still need to clear this extra import fyi @cstns

Reply:
There might be more, my IDE suggested the import and just went with it.